docs/resources/pages_project.md

```diff
@@ -176,27 +176,28 @@ resource "cloudflare_pages_project" "deployment_configs" {
         OTHER_VALUE = "other value"
       }
       secrets = {
-        TURNSTILE_SECRET = var.turnstile_secret
-        TURNSTILE_INVIS_SECRET = var.turnstile_invisible_secret
+        TURNSTILE_SECRET       = "1x0000000000000000000000000000000AA"
+        TURNSTILE_INVIS_SECRET = "2x0000000000000000000000000000000AA"
+
+        kv_namespaces = {
+          KV_BINDING_1 = "5eb63bbbe01eeed093cb22bb8f5acdc3"
+          KV_BINDING_2 = "3cdca5f8bb22bc390deee10ebbb36be5"
+        }
+        durable_object_namespaces = {
+          DO_BINDING_1 = "5eb63bbbe01eeed093cb22bb8f5acdc3"
+          DO_BINDING_2 = "3cdca5f8bb22bc390deee10ebbb36be5"
+        }
+        r2_buckets = {
+          R2_BINDING_1 = "some-bucket"
+          R2_BINDING_2 = "other-bucket"
+        }
+        d1_databases = {
+          D1_BINDING_1 = "445e2955-951a-4358-a35b-a4d0c813f63"
+          D1_BINDING_2 = "a399414b-c697-409a-a688-377db6433cd9"
+        }
+        compatibility_date  = "2022-08-16"
+        compatibility_flags = ["production_flag", "second flag"]
```

Comment:
we should probably use some real flags haha, what about `nodejs_compat` and `streams_enable_constructors`